### ISSUE TITLE
Issues while Training in Colab

### ISSUE THREAD

**Issue by dipamgoswami** [AntonMu/TrainYourOwnYOLO#191]
I have followed the colab tutorial and trained YOLOv3 Model on my own dataset. On Training, I am getting this error: W tensorflow/core/kernels/data/generator_dataset_op.cc:103] Error occurred when finalizing GeneratorDataset iterator: Failed precondition: Python interpreter state is not initialized. The process may be terminated.
	 [[{{node PyFunc}}]]

I can see this error on the sample Colab tutorial provided as well. 
But when I am running the script for detecting objects, its detecting 0 boxes for all the images even when the training loss was less around 80.
I can also see the loss decreasing in Tensorboard.
I think the weights are not getting stored.

Any suggestions how to resolve this issue?

**Comment by parikshit14** [comment]
You are reciving the first error due to the number of images you used in the dataset is not same as compared with that used in the orignal one but till this isn't a problem. Your model will be trained and weights will be stored of that training. 

While the second error is due to **YOUR MODEL IS VERY UNDER TRAINED**
